app/models.py

```diff
@@ -1,23 +1,26 @@
 """Demonstrate all models."""
 
 from werkzeug.security import generate_password_hash, check_password_hash
-from . import db
+from flask_sqlalchemy import SQLAlchemy
+
+
+db = SQLAlchemy()
 
 
 class User(db.Model):
     """Create users table."""
 
     __tablename__ = 'users'
-    uid = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.Integer, primary_key=True)
     email = db.Column(db.String(60))
     username = db.Column(db.String(60))
     first_name = db.Column(db.String(60))
     last_name = db.Column(db.String(60))
     pwd_hash = db.Column(db.String(120))
```

Comment:
It is advisable to use variable names that are easy to relate with.  `pwd_hash` may not be a relatable name choice for most users. You could use `password` instead